kubespawner/objects.py

```diff
@@ -110,13 +113,6 @@ def make_pod_spec(
 
     pod.spec = V1PodSpec()
 
-    security_context = V1PodSecurityContext()
```

Comment:
ooooh, I see what you're saying...

Can init containers have their own security contexts?